tests/integration/test_manylinux.py

```diff
@@ -23,13 +24,15 @@
     'manylinux2010': MANYLINUX2010_IMAGE_ID,
 }
 DOCKER_CONTAINER_NAME = 'auditwheel-test-manylinux'
-PYTHON_IMAGE_ID = 'python:3.5'
+PYTHON_MAJ_MIN = [str(i) for i in sys.version_info[:2]]
```

Comment:
Another approach to this would be to use the `wheel` package which we already depend on. That should provide a better guess at the ABI tag as it handles things like figuring out the allocator, etc.
```
from wheel.pep425tags import get_abbr_impl, get_impl_ver, get_abi_tag
PYTHON_ABI = "{0}{1}-{2}".format(get_abbr_impl(), get_impl_ver(), get_abi_tag()))
```

Reply:
> We can't if I'm not mistaken. sys.executable relates to the python interpreter running the tests while /opt/python/{}/bin'.format(PYTHON_ABI) relates to the python interpreter running in the `manylinux` container.

This is also valid here I think but if there's something better, please correct me.